sstables/sstables.cc

```diff
@@ -2230,6 +2230,18 @@ future<> sstable::move_to_new_dir(sstring new_dir, generation_type new_generatio
     _generation = std::move(new_generation);
 }
 
+future<> sstable::filesystem_storage::activate(const sstable& sst, generation_type gen, delayed_commit_changes* delay_commit) {
```

Comment:
But the sstable is acrive also when its in staging.
While it's there it is not subject to compaction but otherwise it participates in queries etc.

Reply:
Well, it depends on what "active" mean :) I haven't met this term in sstables.cc file, so I didn't suggest that active sstable would really mean something well defined.

Would you suggest the better name, please?

Reply:
I also had an issue with "activate" for the same reason. quarantine, upload, staging are all conditions. a sstable no in those conditions is a normal one. normal = "characterized by that which is considered usual, typical, or routine". so the verb can be **normalize** which means "bring or return to a normal or standard condition or state."

fits very well

Reply:
I'm OK with "normalize" (though my linear algebra professor managed to create incredibly strong "normal == perpendicular" association)